Jswope00/migrate learner docs (#828)
openedx/docs.openedx.org: source/learners/SFD_content_availability.rst

  more information, see :ref:`section_prereq`.
* The course team has made specific course content unavailable after certain
  points in the course, such as after an assignment’s due date has passed or
  after the course end date.

.. _Course Content with Release Dates:

=================================
Course Content with Release Dates
=================================

When course content has release dates, the content is only available on or
after those dates. Many courses release content at a specific time every week.

For information about your specific course, you can check the updates on the
**Course** page, or you can ask questions in the course discussions.

.. _section_prereq:

=================================
Course Content with Prerequisites
=================================

Some courses have content that only becomes available after you achieve a
minimum score in another section. In the course outline, this content appears
with a lock icon and the name of the section that you must complete, or
*prerequisite*.

.. image:: /_images/learners/SFD_Prereq_Locked.png
 :alt: The course outline showing a lock icon next to the name of a section.

.. note::

>   In the edX mobile app, sections that have prerequisites are not visible in

Reviewer: ```suggestion
  In your instance's mobile app, sections that have prerequisites are not visible in
```
